pkg/apiserver/slowquery/queries.go

```diff
@@ -167,12 +144,10 @@ func QuerySlowLogList(db *gorm.DB, req *GetListRequest) ([]Base, error) {
 	}
 
 	order := req.OrderBy
-	if isValidColumnName(order) {
-		if req.DESC {
-			tx = tx.Order(fmt.Sprintf("%s desc", order))
-		} else {
-			tx = tx.Order(fmt.Sprintf("%s asc", order))
-		}
+	if req.DESC {
```

Comment:
ditto. Need to verify the content of `req.OrderBy` in order to avoid SQL injection.